document/CoreFormat.mdk

```diff
@@ -2425,10 +2425,11 @@ This allows tools to include information about the encountered condition that is
 ### General { #exception-General }
 
 An `exception` object describes a runtime exception encountered in the course of executing an analysis tool.
+This includes signals in operating systems such as POSIX.
 
 ### `type` property { #exception-type }
 
-An `exception` object shall contain a property named `type` whose value is a string containing the fully qualified
+An `exception` object should contain a property named `type` whose value is a string containing the fully qualified
```

Comment:
If you're going to include POSIX signals, should this thing be named identifier or something more generic? for C#, this would be populated with the fully qualified type name. For a signal, this could, I guess, be the signal number. And the signal message could be the signal name. @bion?